js/main.js

```diff
@@ -68,7 +70,24 @@ function changeToAddArtPage(event) {
   $divPage4.className = 'page4';
   data.view = 'add-art-page';
   var $addIcons = document.querySelectorAll('.add-icon');
-  console.log($addIcons);
+  var $artPictures = document.querySelectorAll('li > img');
+  $artPictures.forEach(function viewPage(event) {
+    event.addEventListener('click', function hideList(click) {
+      $artList.className = 'hidden';
+      $artViewPage.className = 'view-art-page';
+      var clonedNode = event.parentNode.cloneNode(true);
+      clonedNode.setAttribute('id', 'viewedArt');
+      $displayColumn.appendChild(clonedNode);
+      artDetails(event.id);
+      setTimeout(function () {
+        var artDescription = xhrTwo.response.artObject.plaqueDescriptionEnglish;
+        var DescriptionNode = clonedNode.lastChild;
+        DescriptionNode.innerHTML = '<b>Description:</b> ' + artDescription;
+
+      }, 1000);
```

Comment:
You should be doing this in the `load` event handler for the request, not using a setTimeout. Using a `setTimeout` to wait for data is **_never_** a good idea